news/_posts/2020-08-29-Polymake.jl-0.5-release.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Polymake.jl-0.5 release
+author: Marek Kaluba
+---
+`Polymake.jl` v0.5.0 has been released!<br/>
+
+The main and show-stealing reason to bump the major version is the transition of `Polymake.jl` from `BinaryBuilder` dependencies to modern Julia artifacts system. Thanks to the transition there is literally no compilation happening during `add Polymake` and all our libraries should be shining and working with each other.<br/>
+
+A huge round of applause is due to @benlorenz, who single-handedly cross-compiled `perl` for MacOsX on a MIPS router dev-board to make this happen. Without him, nobody would dare to look into the eyes of the beast.<br/>
```

Comment:
```suggestion
A huge round of applause is due to @benlorenz, who single-handedly cross-compiled `perl` for MacOsX on a MIPS router dev-board to make this happen. Without him, nobody would dare to look into the eyes of the beast.

<br/>
```
(Either this, or doubling the `<br/>` will do, in order to insert a visual blank line)